src/Contract/Config/Formatter/MermaidJsConfig.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Contract\Config\Formatter;
+
+use Qossmic\Deptrac\Contract\Config\Layer;
+
+final class MermaidJsConfig implements FormatterConfigInterface
+{
+    private string $name = 'mermaidjs';
+
+    private string $direction = 'TD';
```

Comment:
In the `yaml` config, you allow to modify this parameter, but I do not see an equivalent way to do this in the `php` config. Can you make sure the feature set is equivalent between the two?

Reply:
Should be now equivalent.